src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -133,9 +133,18 @@ std::shared_ptr<ov::npuw::ICompiledModel> ov::npuw::ICompiledModel::create(
     auto use_llm_key = ov::intel_npu::npuw::llm::enabled.name();
     if (properties.count(use_llm_key) && properties.at(use_llm_key).as<bool>() == true) {
         LOG_INFO("ov::npuw::LLMCompiledModel will be created.");
-        compiled_model = std::make_shared<ov::npuw::LLMCompiledModel>(model, plugin, properties);
+        // Drop CACHE_DIR from the config
+        // If it's present we will be utilizing LLMCompiledModel's import
+        // and not the underlying models and submodels
+        auto config = properties;
+        config.erase(ov::cache_dir.name());
+        compiled_model = std::make_shared<ov::npuw::LLMCompiledModel>(model, plugin, config);
```

Comment:
this is strange, why should you drop cache_dir when you enter the LLMCompiledModel - as that one does support the caching. Probably what is to be removed is this automatic translation from `CACHE_DIR` to `NPUW_CACHE_DIR` if it is done by the LLMCompiledModel. Is it?

Reply:
The point is that caching is done a lever higher by OV itself. Here I just prevent the plugin (CPU/NPU) from creating an additional cached blobs (since it's already inside our own blob).

Reply:
ok so the idea here is not to dispatch CACHE_DIR to the downstream models? I believe this is what NPUW's CompiledModel should handle - please file a task to clear up this.